pkg/bpf/bpf_linux.go

```diff
@@ -595,7 +595,10 @@ recreate:
 			flags,
 		)
 		if redo {
-			ObjClose(fd)
+			err := ObjClose(fd)
+			if err != nil {
+				return fd, false, err
+			}
```

Comment:
@pchaigno @NikAleksandrov I don't know if this change was here when you left a review but does this look good to you?

Reply:
It probably doesn't matter if we fail to close the fd at this stage, ignoring the error (`_ = ObjClose(fd)`) would seem fine to me.

Reply:
Thanks

Reply:
@yulng I believe Quentin was suggesting to keep this code as it was before.

Reply:
I agree with Quentin, I think we can ignore it here.

Reply:
@yulng can we revert this change?

Reply:
ok,done
Thanks